src/main/java/javax/validation/ConstraintValidatorContext.java

```diff
@@ -648,6 +749,13 @@
 		interface ContainerElementNodeBuilderDefinedContext {
 
 			/**
+			 * Marks the node as being in an {@code Iterable} or a {@code Map}.
+			 *
+			 * @return a builder representing iterable details
+			 */
+			NodeContextBuilder inIterable();
```

Comment:
What's the reason for adding this? It's not on the other `DefineContext` types, they say "a node whose context is known (i.e. index, key and isInIterable)".